src/main/java/edu/hawaii/its/api/service/OotbActiveUserProfileService.java

```diff
@@ -0,0 +1,74 @@
+package edu.hawaii.its.api.service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import jakarta.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.hawaii.its.api.type.OotbActiveUserProfile;
+
+@Service
+public class OotbActiveUserProfileService {
+
+    @Autowired
+    private OotbGroupingPropertiesService ootbGroupingPropertiesService;
+    private Map<String, OotbActiveUserProfile> users = new HashMap<>();
+    private String ootbActiveUserProfile = "ADMIN";
+
+    @PostConstruct void init() {
+        initUsers();
+        updateActiveDefaultUser(ootbActiveUserProfile);
+    }
+
+    private void initUsers() {
+        users.put("USER", new OotbActiveUserProfile.Builder()
+                .uid("user0123")
+                .uhUuid("11111111")
```

Comment:
On second look at this PR, I'm confused about having this code for the active user to be in the API side. The UI side has always taken control of the user authentication/authorization. If possible, it would be best to keep all the user code in the UI.